cmd/build.go

```diff
@@ -42,7 +42,6 @@ type buildCmd struct {
 	isolation       string
 	network         string
 	platform        string
-	homeVol         string
```

Comment:
What was the use case of the homeVol before? I see the code now always creates a new homeVol as SharedContextDir. I'm just curious if some scenarios won't work.